internal/core/command/main.go

```diff
@@ -111,6 +109,11 @@ func MessageBusBootstrapHandler(ctx context.Context, wg *sync.WaitGroup, startup
 			return false
 		}
 	}
+	if configuration.MessageQueue.External.Enabled {
```

Comment:
I think this check should be within `if configuration.RequireMessageBus` scope since receiving external message doesn't make sense if we are not connected to the internal message bus to send them to the device services